server/internal/packetio.go

```diff
@@ -381,12 +381,13 @@ func (cw *compressedWriter) Flush() error {
 	// https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_basic_compression_packet.html
 	// suggests a MIN_COMPRESS_LENGTH of 50.
 	minCompressLength := 50
+	zlibCompressDefaultLevel := 6
```

Comment:
Why "6" here?

Reply:
This is MySQL's choice https://github.com/pingcap/tidb/issues/47278#issuecomment-1734735799